src/verifact_agents/evidence_hunter.py

```
@@ -10,46 +15,194 @@
+    with open(path, "r", encoding="utf-8") as f:
+        return [
+            line.strip()
+            for line in f
+            if line.strip() and not line.strip().startswith("#")
+        ]
+
+class EvidenceHunter:
+
+    def __init__(self,trust_sources_path: str="data\\trust_sources.txt"):
+        """Initialize the evidence hunter with a claim.
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)